src/css/index.css

```diff
@@ -0,0 +1,160 @@
+@import url('https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&family=Poppins:wght@400;700&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200');
+
+/* GOOGLE ICONS */
+
+.material-symbols-outlined {
+  font-variation-settings: 'FILL' 0, 'wght' 700, 'GRAD' 0, 'opsz' 48;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+}
+
+*:focus {
+  outline: none;
+}
+
+body {
+  background-color: #f6f6f6;
+  font-family: 'Poppins', sans-serif;
+}
+
+main {
+  display: flex;
+  justify-content: center;
+}
+
+.container {
+  width: 700px;
+  min-width: 290px;
+  margin: 30px;
+  display: flex;
+  flex-direction: column;
+  background-color: #fff;
+  box-shadow: 1px 2px 10px 1px #5d5b5b;
+}
+
+li {
+  border-bottom: 1px solid lightgray;
+}
+
+/* LIST HEADER */
+
+.list-header {
+  display: flex;
+  flex-direction: column;
+  gap: 5px;
+  list-style: none;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  padding: 15px;
+  font-size: 20px;
+  border-bottom: 2px solid lightgray;
+}
+
+#addToDoForm {
+  display: flex;
+  justify-content: space-between;
+  padding: 15px;
+}
+
+#toDoInput {
+  width: 90%;
+  font-size: 18px;
+  font-weight: 500;
+  border: none;
+  color: #a8a8a8;
+  font-style: italic;
+}
```

Comment:
I kindly suggest avoiding using IDs to style elements in CSS this is for specificity reasons, for further details I suggest checking the [Reasons not to use IDs in CSS](https://dev.to/clairecodes/reasons-not-to-use-ids-in-css-4ni4) article.

Instead set a class to the element you want to style.